live-examples/js-examples/string/string-length.html

```diff
@@ -0,0 +1,7 @@
+<pre>
+<code id="static-js">const str = 'Life, the universe and everything. Answer:';
+  
+console.log(str + ' ' + str.length);
+// expected result: 42
```

Comment:
"expected output", not "expected result". Sorry to be picky, but we might as well be consistent here.

Also the output is `"Life, the universe and everything. Answer: 42"`, not `42`.